geonode/thumbs/utils.py

```diff
@@ -261,3 +261,63 @@ def fetch_wms(url: str, max_retries: int = 3, retry_delay: int = 1):
             break
 
     return image
+
+
+def epsg_3857_area_of_use():
```

Comment:
I would set the returned values as constants (i.e. populate the constants from this method at import time), without calculating them each time.